MetaMask/metamask-mobile
# [5.3.0][FIX] Prevent crash when funds warning is pressed

**Description**
The app is crashing when we do not have enough funds to do a transaction on testnet

**Proposed Solution**
The app will redirect to the MetaMask faucet when it's testnet
The app will redirect to onRamp with the crypto we are trying to transact

**Code Impact**
Medium, was changed in all transaction confirmations screens/modals:
* Confirm screen of send flow
* Modal of transaction Review
* Modal of transaction approval 

**Test Cases**
(For mainnet, avax and rinkeby)
Case 1:
* Press the warning on the confirm screen of send flow
Case 2:
* Press the warning on transaction modal of deeplinks
Case 3:
* Press the warning on transaction modal of webview

**Screenshots/Recordings**
Bug: 
http://recordit.co/wpPgIUiPcG

Solution:
https://recordit.co/KR5KHCtYOV

**Checklist**


**Issue**

Progresses #4177
